src/main/java/Duke.java

```diff
@@ -1,10 +1,368 @@
+import java.io.*;
+import java.util.Scanner;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+
+// level 9 commits branch
+// sample level 9 change
+
 public class Duke {
+
+    public static int counter = 0; // this has to be a global variable
+    public static Task[] tasks = new Task[100]; // another global variable
+
+    /*
```

Comment:
Consider removing outdated code since github is already doing version control for you.